StripeFinancialConnections/StripeFinancialConnections/Source/Native/NetworkingLinkStepUpVerification/NetworkingLinkStepUpVerificationDataSource.swift

```diff
@@ -50,7 +50,8 @@ final class NetworkingLinkStepUpVerificationDataSourceImplementation: Networking
             consumerSession: nil,
             apiClient: apiClient,
             clientSecret: clientSecret,
-            analyticsClient: analyticsClient
+            analyticsClient: analyticsClient,
+            isTestmode: manifest.isTestMode
```

Comment:
```
isTestmode: manifest.isTestMode
```

one uses lowercase "m" (isTestmode) the other uses upper case "M" (manifest.isTestMode) - is that expected?

Reply:
Whoops - good catch!